### ISSUE TITLE
Cannot bind to style when binding to evets

### ISSUE THREAD

**Issue by rpedretti** [aspnet/Blazor#784]
I'm using Blazor 0.3.0 and whenever I bind to an event any bind to 'style' breaks.

I have a sample app at [https://github.com/rpedretti/BlazorApp](https://github.com/rpedretti/BlazorApp) on branch version_0_3_0 that the bug can be reproduced, but I guess it' happens in any project

Given a simple tag like

```html
<p onmouseover="@ViewModel.OnComponentHover"
  style="background: @ViewModel.ParentBgColor;" />
```

The compiler throws 

> Error Component attributes do not support complex content (mixed C# and markup). Attribute: 'style', text 'background:ViewModel.ParentBgColor;

and even the intellisense breakes:
![image](https://user-images.githubusercontent.com/675601/39705115-d74273bc-51e3-11e8-83a8-39cd8511f4e2.png)

By removing any of the properties the project compiles beautifully and intellisense works again.

![image](https://user-images.githubusercontent.com/675601/39705170-fc444618-51e3-11e8-89e9-c9fca3884ffb.png)

**Comment by SteveSandersonMS** [comment]
Actually I no longer thing this is fixed in 0.4.0. Needs more investigation.

**Comment by SteveSandersonMS** [comment]
No, the repro info above is sufficient. Thanks for reporting it! We'll investigate.

**Comment by rynowak** [comment]
I'll take a look at this.

**Comment by rynowak** [comment]
This should be fixed in dev as part of the upcoming 0.4 release

**Comment by vargonian** [comment]
I'm using 0.5.1 and still having this issue.

**Comment by danroth27** [comment]
@vargonian Please open an issue with your steps for reproducing the issue and we'll take a look.
